### PR TITLE
Fea/change hamburger icon navbar

### PR DESCRIPTION
**Pull request by digvijay-gurav**

### CODE REVIEW

**Comment by digvijay-gurav** [comment]
Remove hamburger

File: LESS/navigation.less [hunk 1]
```diff
@@ -697,3 +697,33 @@ body.canvas-menu.mini-navbar nav.navbar-static-side {
   display: block;
   width: 220px;
 }
+.navBarMenuToggle {
```
Comment (neerajdembla212) [inline LESS/navigation.less:700]: please change the class names to kebab case as standardly used in theme. eg `navbar-menu-toggle`

File: LESS/navigation.less [hunk 2]
```diff
@@ -697,3 +697,33 @@ body.canvas-menu.mini-navbar nav.navbar-static-side {
...
+  left: 6px;
+}
+.mini-navbar .navMenuToggleArrow {
+  width: 0; height: 0; 
+  border-top: 5px solid transparent; 
+  border-left: 8px solid #734CF5;
+  border-bottom: 5px solid transparent;
+  border-right: 0;
+  display: block;
+  position: absolute;
```
Comment (neerajdembla212) [inline LESS/navigation.less:726]: Why do we need to add `position: absolute` ? current hamburger works fine without this property .

**Review by neerajdembla212 — CHANGES_REQUESTED** [review]

**Comment by neerajdembla212** [comment]
Closing this PR duplicate PR for https://github.com/neerajdembla212/neerajdembla212.github.io/pull/11

**Pull request closed without merging**